students/ahyounglee/users/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+from django.core.validators import RegexValidator
+
+
+class User(models.Model):
```

Comment:
사용하지 않는 모듈은 제거해 주시고, class와의 간격은 1줄로 해주세요.
```suggestion
from django.db import models

class User(models.Model):
```

Reply:
수정했습니다.